Cassandra chart install fails with "spec.selector: required value" error

With helm version `helm version v0.1.0+c600ce8`, I got the following error installing Cassandra:

```
pentagon:Applications paul$ helm search cassandra
        cassandra (cassandra 0.2.0) - The Apache Cassandra Project.
pentagon:Applications paul$ helm info cassandra
Name: cassandra
Home: http://cassandra.apache.org/
Version: 0.2.0
Description: The Apache Cassandra Project.
Details: This package deploys Cassandra to kubernetes.

pentagon:Applications paul$ helm install cassandra
---> No chart named "cassandra" in your workspace. Fetching now.
---> Running `kubectl create -f` ...
service "cassandra" created
The ReplicationController "cassandra" is invalid.
spec.selector: required value
[ERROR] Failed to upload manifests: exit status 1
```

--- a/cassandra/Chart.yaml
+++ b/cassandra/Chart.yaml
@@ -1,6 +1,6 @@
 name: cassandra
 home: http://cassandra.apache.org/
-version: 0.2.0
+version: 0.2.1
 description: The Apache Cassandra Project.
 maintainers:
   - Matt Fisher <mfisher@deis.com>

--- a/cassandra/manifests/cassandra-rc.yaml
+++ b/cassandra/manifests/cassandra-rc.yaml
@@ -7,7 +7,13 @@ metadata:
     heritage: helm
 spec:
   replicas: 1
+  selector:
+    provider: cassandra
   template:
+    metadata:
+      name: cassandra
+      labels:
+        provider: cassandra
     spec:
       containers:
       - name: cassandra

